pkg/secrethub/client.go

```diff
@@ -13,7 +13,7 @@ import (
 type Client interface {
 	AccessRules() AccessRuleService
 	Accounts() AccountService
-	Auth() AuthService
```

Comment:
If it's private, should it be part of the interface or can it just be a function on the concrete implementation of the interface?

Reply:
Also the return type is public, should that be private too?